coalib/settings/ConfigurationGathering.py

```diff
@@ -385,6 +385,19 @@ def gather_configuration(acquire_settings,
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings,
                                               log_printer)
+
+    exclude = False
+    for x in arg_list:
+        if x == '-ex' or x == '--exclude-sections':
+            exclude = True
+        elif exclude and '-' not in x:
+            if x in sections:
+                del sections[x]
+            else:
+                logging.warning('The {} section doesn\'t exists'.format(x))
+        elif '-' in x:
```

Comment:
Because we can have multiple sections that a user can exclude at a run. So I need to know where the list is ending